dpctl/__init__.pxd

```diff
@@ -1,4 +1,4 @@
-#===------------- __init__.pxd - dpctl module --------*- Cython -*----------===#
+# ===------------ __init__.pxd - dpctl module --------*- Cython -*----------===#
```

Comment:
It is not necessary to fix number of #.
Possibly @diptorupd you are using old version of black. Use `20.8b1` as pointed in [CONTRIBUTING.md](https://github.com/IntelPython/dpctl/blob/gold/2021/CONTRIBUTING.md). Conda provides version `19.10b0`. Pip provides `20.8b1`.

Reply:
Thanks!